app/styles/_shame.scss

```diff
@@ -0,0 +1,45 @@
+/* ==========================================================================
+   #SHAME
+   Dirty temporary css for demo purposes.
+   Some parts will be reused, but most of them will not.
+   ========================================================================== */
+
+th {
+  background-color: white;
+  font-weight: bold;
+}
+
+tr:nth-child(odd) {
+  background-color: #f2f2f2;
+}
+
+.triple table {
+  border-collapse: collapse;
+}
+
+.triple table, .triple th, .triple td {
+  border: 1px solid black;
+}
+
+/* ==================================
+   #NON-EDITABLE
+   ================================== */
+
+/* Variables
+   ========================================================================== */
+
+$au-non-editable-background    : $au-neutral-lightest !default;
+
+.au-c-form--non-editable {
```

Comment:
Is there a `.au-c-form` component? If not we should not use a modifier for this.

Reply:
it is in shame because I think we can use it at some point, but still thinking. Can I put this point on the to do before we fix it here? It requires some more thinking.

Reply:
No need to fix. In shame you can do this :-) Just mentioning the component/modifier issue.

Reply:
Perfecto!